src/device/dcd.h

```diff
@@ -66,7 +66,7 @@ typedef struct TU_ATTR_ALIGNED(4)
 
     // USBD_EVT_XFER_COMPLETE
     struct {
-      uint8_t  ep_addr;
+      uint8_t  ep_addr; ///< 0xFF signifies that the transfer was aborted.
```

Comment:
I'm not seeing the utility in creating a special ep_addr value to indicate that a non-unknown EP number has had an aborted transfer.  Why not, say, set the len to zero (and keep the EP number as-is)?

Reply:
yeah, I think we need to spend a bit more time to evaluate how to address the on-going transfer. I plan to try it out with nrf52 to see what is the best approach.

Reply:
length zero can indicate a ZLP.

We could create a separate event type, EVENT_INVALID?

Reply:
Oh. I'm wrong. We can just get rid of the event entirely, no need to requeue it. The modification was based on the earlier revision of the code where it simplified life. :(

Reply:
(patch pushed to branch, so marking this as resolved.)